managers/oclc_catalog.py

```diff
@@ -96,17 +96,24 @@ def _get_other_editions(self, oclc_number: int, offset: int=0):
             )
 
             if other_editions_response.status_code != 200:
-                logger.warning(f'OCLC other editions request failed with status {other_editions_response.status_code}')
+                logger.warning(
+                            f'OCLC search bibs request for OCLC no {oclc_number} failed '
+                            f'with status {other_editions_response.status_code}')
+                try:
+                            oclc_error_type = other_editions_response.json()["type"]
```

Comment:
I think we can do get('type') here instead to just guard.

Reply:
I think json() could still throw a JSONDecodeError before the get() is run on type, but it could be split out such that the exception is narrowed to not include both the JSONDecodeError and the KeyError here - do you think that would be better?

Reply:
(The possibilities including: the response not having valid JSON, and the response having valid JSON but it might not contain a type)

Reply:
But we aren't handling the TypeError right? I think using get would prevent the KeyError.